tests/array_extensibility_test.py

```diff
@@ -573,5 +573,10 @@ def wrap_array(arg):
     self.assertAllClose(actual, expected, atol=0, rtol=0)
 
 
+@jtu.with_config(jax_disable_jit=True)
+class JaxArrayTestsNoJit(JaxArrayTests):
```

Comment:
This is here for future reference?

Reply:
This is here because it results in all tests from `JaxArrayTests` to also be run with `jax_disable_jit=True`. This led to ~140 failures before the other changes in this PR.